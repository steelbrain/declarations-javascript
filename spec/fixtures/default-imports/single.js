import moduleName from 'file-moduleName'
moduleName()
