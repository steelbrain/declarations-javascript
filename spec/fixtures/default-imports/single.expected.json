[
  {
    "name": null,
    "position": {
      "start": {
        "line": 1,
        "column": 23
      },
      "end": {
        "line": 1,
        "column": 40
      }
    },
    "source": {
      "name": null,
      "filePath": "file-moduleName",
      "position": {
        "start": {
          "line": 1,
          "column": 23
        },
        "end": {
          "line": 1,
          "column": 40
        }
      }
    }
  },
  {
    "name": "moduleName",
    "position": {
      "start": {
        "line": 2,
        "column": 0
      },
      "end": {
        "line": 2,
        "column": 10
      },
      "identifierName": "moduleName"
    },
    "source": {
      "name": null,
      "filePath": null,
      "position": {
        "start": {
          "line": 1,
          "column": 7
        },
        "end": {
          "line": 1,
          "column": 17
        },
        "identifierName": "moduleName"
      }
    }
  }
]